perfkitbenchmarker/benchmarks/fio_benchmark.py

```diff
@@ -20,24 +20,62 @@
 
 import json
 import logging
+import posixpath
+import re
+
+import jinja2
 
 from perfkitbenchmarker import data
 from perfkitbenchmarker import flags
 from perfkitbenchmarker import vm_util
 from perfkitbenchmarker.packages import fio
 
+LOCAL_JOB_FILE_NAME = 'fio.job'  # used with vm_util.PrependTempDir()
+REMOTE_JOB_FILE_PATH = posixpath.join(vm_util.VM_TMP_DIR, 'fio.job')
+DEFAULT_TEMP_FILE_NAME = 'fio-temp-file'
+MAX_FILE_SIZE_GB = 100
+DISK_USABLE_SPACE_FRACTION = 0.9
+
+
+# This dictionary maps scenario names to dictionaries of fio settings.
+SCENARIOS = {
+    'sequential_write': {
+        'name': 'sequential_write',
+        'rwkind': 'write',
+        'blocksize': '512k'
+    },
+    'sequential_read': {
+        'name': 'sequential_read',
+        'rwkind': 'read',
+        'blocksize': '512k'
+    },
+    'random_write': {
+        'name': 'random_write',
+        'rwkind': 'randwrite',
+        'blocksize': '4k'
+    },
+    'random_read': {
+        'name': 'random_read',
+        'rwkind': 'randread',
+        'blocksize': '4k'
+    }
+}
+
 
 FLAGS = flags.FLAGS
 
-flags.DEFINE_string('fio_benchmark_filename', 'fio_benchmark_file',
-                    'scratch file that fio will use')
-flags.DEFINE_string('fio_jobfile', 'fio.job', 'job file that fio will use')
-flags.DEFINE_integer('memory_multiple', 10,
-                     'size of fio scratch file compared to main memory size.')
+flags.DEFINE_string('fio_jobfile', None,
```

Comment:
I don't think it should look for an absolute directory, actually. I'd like it to behave the same way as other command-line utilities, so the path is relative to your working directory. However, when you pass it a job file, it always copies it to its temporary directory.